packages/components/src/components/Loader/Analysis/style.module.scss

```diff
@@ -0,0 +1,88 @@
+.divider {
+  border-color: rgba(0, 0, 0, 0.12);
+  position: relative;
+  bottom: 20%;
+}
+
+.dividerDiv {
+  flex-grow: 1;
+  margin-left: 10px;
+}
+
+.box {
+  width: 260px;
+  display: flex;
+  height: 40px;
+
+  .keyword {
+    flex: 0 0 auto;
+    padding-right: 5;
+  }
+}
+
+.textBox {
+  max-width: 220px;
+  color: rgba(0, 0, 0, 0.85);
+  height: 22px;
+  border: 1px #d9d9d9 solid;
+  justify-content: center;
+  display: flex;
+  padding: 1px 8px 1px 8px;
+  align-items: center;
+  border-radius: 2px;
+  background-color: #fafafa;
+
+  .text {
+    font-size: 12px;
+    font-weight: 400;
+    font-family: 'Roboto';
```

Comment:
font family should use global css variables 「--font-family-code」in components/src/pages/Overall/index.module.scss